### PR TITLE
Rt updates

### LINKED ISSUE #302: Adding RT metrics to reports
**Is your feature request related to a problem? Please describe.**
The GTFS Quality team uses some key metrics regarding agencies to see if their RT systems are doing a good job or it's something we want them to fix.

You can find an example dashboard here: https://dashboards.calitp.org/dashboard/133-individual-agency-completeness?text=Bay%20Area%20511%20Muni%20Schedule

**Describe the solution you'd like**
We want to add some of these charts to the reports.
It should look something like this:
image

### PR DESCRIPTION
**Pull request by vevetron**
# Description

Added some datasets and charts to agencies reports if they have RT.

Couldn't figure out how to use Siuba for everything so wrote custom sql / pandas when it made sense.

Extended the charting to handle all these new scenarios.

Resolves #302 

## Type of change

- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?
It's been build, but not gone through extensive validation testing.

## Screenshots (optional)
image

## Followup
Check to see if appropriate agencies have data.  Validation testing?

### CODE REVIEW

**Comment by edasmalchi** [comment]
Seems like the y-axis label on the message age chart is getting clipped, do we think we can squeeze that in/add units (is it minutes?). Could do something like "Avg Trip Median [VP/TU] Message Age, minutes"

Also might suggest adding a brief explanation of what these charts signify, in the same style of the "About..." section that exists for the service hours and identifier change charts below

**Comment by vevetron** [comment]
Added about sections and shortened y axises.

image

image

**Comment by vevetron** [comment]
The way the package-lock.json got updated seems a little funny.

**Comment by vevetron** [comment]
"Average Daily Median"

File: templates/report.html.jinja [hunk 1]
```diff
@@ -106,7 +108,89 @@
...
+                 data-tooltip-value-label="percent"
+                 data-chart-type="area"
+                 data-chart-collabel="vehicle_position_completeness">
+            </div>
+        </div>
+    </div>
+        <div class="flex-1 responsive-prose">
+            <h2>About the Median Message Age&nbsp;Charts</h2>
+            <ul>
+            <li>These charts show the average median age of trip updates and vehicle position messages over time.</li>
```
Comment (edasmalchi) [inline templates/report.html.jinja:155]: capitalize Trip Updates + Vehicle Positions for consistency w/ above?

Comment (vevetron) [reply templates/report.html.jinja]: Done

**Review by edasmalchi — APPROVED** [review]
looks great, thanks @vevetron!!

**Pull request merged**